Tests/DoctorPrettyTests/DoctorPrettyTests.swift

```diff
@@ -192,6 +192,28 @@ class DoctorPrettyTests: XCTestCase {
             ].joined(separator: "\n"), doc: doc)
     }
 
+    func testLargeDocument() {
+        let doc = (1...65)
```

Comment:
so this is my "large document" test. before the `fits` unrolling, I couldn't get this document to layout using only a paltry 30 `"foo"` strings. now we can go higher than 65, but it takes about a second to layout this document.

I'm sure there are bigger wins to be had.